src/diffusers/pipelines/wuerstchen/modeling_wuerstchen_prior.py

```diff
@@ -14,16 +14,27 @@
 # limitations under the License.
 
 import math
+from typing import Dict, Union
 
 import torch
 import torch.nn as nn
 
 from ...configuration_utils import ConfigMixin, register_to_config
+from ...loaders import UNet2DConditionLoadersMixin
+from ...models.attention_processor import (
+    ADDED_KV_ATTENTION_PROCESSORS,
+    CROSS_ATTENTION_PROCESSORS,
+    AttentionProcessor,
+    AttnAddedKVProcessor,
+    AttnProcessor,
+)
 from ...models.modeling_utils import ModelMixin
 from .modeling_wuerstchen_common import AttnBlock, ResBlock, TimestepBlock, WuerstchenLayerNorm
 
 
-class WuerstchenPrior(ModelMixin, ConfigMixin):
+class WuerstchenPrior(ModelMixin, ConfigMixin, UNet2DConditionLoadersMixin):
+    UNET_NAME = "prior"
```

Comment:
```suggestion
    unet_name = "prior"
```

Reply:
@patrickvonplaten  for the `UNet2DConditionLoadersMixin` i also need it as small camel case?